.gitignore

```diff
@@ -3,6 +3,7 @@ test/mock_api_server/__pycache__
 
 vendor
 .idea
+.vscode
```

Comment:
Intentional change?

Reply:
yes, I use vscode to develop. This may help others using vscode.